package.json

```diff
@@ -1,142 +1,5 @@
-    "eslint-plugin-flowtype": "^2.29.1",
-    "eslint-plugin-import": "^2.2.0",
-    "eslint-plugin-jsx-a11y": "^2.2.3",
-    "eslint-plugin-react": "^6.5.0",
-    "expect.js": "^0.3.1",
-    "flow-bin": "^0.72.0",
-    "json-loader": "^0.5.4",
-    "jss-cache": "^2.0.0",
-    "jss-camel-case": "^6.1.0",
-    "jss-compose": "^4.0.0",
-    "jss-default-unit": "^7.0.0",
-    "jss-expand": "^4.0.0",
-    "jss-extend": "^6.2.0",
-    "jss-global": "^2.0.0",
-    "jss-isolate": "^4.0.0",
-    "jss-nested": "^5.0.0",
-    "jss-props-sort": "^5.0.0",
-    "jss-vendor-prefixer": "^6.0.0",
-    "karma": "^1.3.0",
-    "karma-benchmark": "^0.6.0",
-    "karma-benchmark-reporter": "^0.1.1",
-    "karma-browserstack-launcher": "^1.1.1",
-    "karma-chrome-launcher": "^2.0.0",
-    "karma-coverage": "^1.1.1",
-    "karma-mocha": "^1.3.0",
-    "karma-mocha-reporter": "^2.2.1",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^1.8.0",
-    "lint-staged": "^3.2.2",
-    "mocha": "^3.2.0",
-    "pre-commit": "^1.1.3",
-    "prettier": "^1.10.2",
-    "rimraf": "^2.5.4",
-    "rollup": "^0.58.2",
-    "rollup-plugin-babel": "^3.0.4",
-    "rollup-plugin-commonjs": "^9.1.3",
-    "rollup-plugin-node-resolve": "^3.3.0",
-    "rollup-plugin-replace": "^2.0.0",
-    "rollup-plugin-size-snapshot": "^0.4.1",
-    "rollup-plugin-uglify": "^3.0.0",
-    "webpack": "^2.3.3",
-    "zen-observable": "^0.6.0"
-  },
-  "scripts": {
-    "all": "npm run lint && npm run typecheck && npm run test && npm run build",
-    "build": "npm run clean && npm run build:lib && npm run build:umd && npm run build:tests",
-    "build:lib": "babel src --out-dir lib && npm run build:flow",
-    "build:umd": "rollup -c",
-    "build:tests": "cross-env NODE_ENV=test webpack tests/index.js tmp/tests.js",
-    "build:flow": "echo \"// @flow\n\nexport * from '../src';\" > lib/index.js.flow",
-    "clean": "rimraf '{lib,dist,tmp}/*'",
-    "typecheck": "flow check --max-warnings=0",
-    "lint": "eslint ./src ./tests ./benchmark ./*.js && npm run typecheck && npm run format",
-    "lint:staged": "lint-staged && npm run typecheck",
-    "format": "prettier --write */**/*",
-    "test": "cross-env NODE_ENV=test karma start --single-run",
-    "test:watch": "cross-env NODE_ENV=test karma start",
-    "posttest": "cross-env SNAPSHOT=match rollup -c && [ -z \"$TRAVIS\" ] || codecov",
-    "codecov": "codecov",
-    "bench": "cross-env BENCHMARK=true karma start --single-run",
-    "prepublishOnly": "npm run all",
-    "postinstall": "node -e \"console.log('\\u001b[35m\\u001b[1mLove JSS? You can now support us on open collective:\\u001b[22m\\u001b[39m\\n > \\u001b[34mhttps://opencollective.com/jss/donate\\u001b[0m')\""
-  },
-  "dependencies": {
-    "is-in-browser": "^1.1.3",
-    "symbol-observable": "^1.1.0",
-    "warning": "^3.0.0"
-  },
-  "files": [
-    "dist",
-    "lib",
-    "src",
-    "flow-typed",
-    "readme.md",
-    "changelog.md",
-    "LICENSE"
-  ],
-  "lint-staged": {
```

Comment:
`lint-staged` should also be run from the root

Reply:
Yeah, I still need to move all the dev deps and setup scripts in the root package.